package.json

```diff
@@ -22,10 +24,14 @@
   },
   "devDependencies": {
     "prettier": "^1.15.3",
+    "rollup": "^2.7.2",
+    "rollup-plugin-auto-external": "^2.0.0",
+    "rollup-plugin-typescript2": "^0.27.0",
     "source-map-support": "^0.5.9",
+    "tslib": "^1.11.1",
```

Comment:
How come we need to list this as a dependency now?

Reply:
Thanks for pointing this out. It is an unnecessary dependency as it is an implicit dependency of the rollup plugin.